tensorflow_lite_support/python/test/task/audio/core/tensor_audio_test.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for tensor_audio."""
+import unittest
+from unittest import mock
+
+import numpy as np
+from numpy import testing
+
+from tensorflow_lite_support.python.task.audio.core import tensor_audio
+from tensorflow_lite_support.python.task.audio.core import audio_record
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+from tensorflow_lite_support.python.test import test_util
+
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_CppAudioBuffer = _pywrap_audio_buffer.AudioBuffer
+
+_CHANNELS = 1
+_SAMPLE_RATE = 16000
+_SAMPLE_COUNT = 15600
+
+
+class TensorAudioTest(unittest.TestCase):
+  def setUp(self):
+    super().setUp()
+    self.test_audio_path = test_util.get_test_data_path('speech.wav')
+    self.test_tensor_audio = tensor_audio.TensorAudio(
+      _CppAudioFormat(_CHANNELS, _SAMPLE_RATE), _SAMPLE_COUNT)
+
+  def test_create_from_wav_file(self):
+    # Loads TensorAudio object from WAV file.
+    tensor = tensor_audio.TensorAudio.create_from_wav_file(
+      self.test_audio_path, _SAMPLE_COUNT)
+    tensor_audio_format = tensor.format
+
+    self.assertEqual(tensor_audio_format.channels, _CHANNELS)
+    self.assertEqual(tensor_audio_format.sample_rate, _SAMPLE_RATE)
+    self.assertEqual(tensor.sample_count, _SAMPLE_COUNT)
+    self.assertIsInstance(tensor.data, _CppAudioBuffer)
```

Comment:
Please assert some values in the buffer to ensure the WAV file is loaded correctly.